dom/nodes/NodeList-Iterable.html

```diff
@@ -34,4 +36,24 @@
             assert_equals(node.getAttribute('id'), ids[idx++]);
         }
     }, 'NodeList is iterable via for-of loop.');
+
+    test(function() {
+        assert_array_equals(Object.keys(paragraphs), ['0', '1', '2', '3', '4']);
+    }, 'NodeList responds to Object.keys correctly');
+
+    test(function() {
+        const container = document.getElementById('live');
+        const nodeList = container.childNodes;
+
+        let total = 0;
+        for (const el of nodeList) {
+            ++total;
+            assert_equals(el.localName, "b");
+            if (total < 3) {
+                container.appendChild(document.createElement("b"));
+            }
+        }
+
+        assert_equals(total, 5);
```

Comment:
Can we also confirm the order in which elements appear?
e.g. we can add class/id/title on each `b` to identify each b's.